src/type-generator.ts

```diff
@@ -36,17 +36,42 @@ export class TypeGenerator {
     );
   }
 
-  buildColumnType(column: pgStructure.Column): IColumnType {
-    const columnType = (column.type.internalName ??
+  getEnumTypeName(enumType: pgStructure.Type) {
+    return enumType.fullName.split('.').join('_');
+  }
+
+  buildColumnTypeMapping(column: pgStructure.Column) {
```

Comment:
Should this be named something more like "getMappedColumnType"?

Reply:
Yea, that's seems like a better name